D2L.Hypermedia.Siren/SirenEntity.cs

```diff
@@ -158,6 +160,33 @@ public class SirenEntity : ISirenEntity {
 
 		}
 
+		string ISirenSerializable.ToJson() {
+			StringBuilder sb = new StringBuilder();
+			StringWriter sw = new StringWriter( sb );
+			using( JsonWriter writer = new JsonTextWriter( sw ) ) {

```

Comment:
I'm not in VisualStudio, so just doing this in my head, but `new JsonTextWriter( sw ) { CloseOutput = true }` might be safer if you're relying on it to close the `StringWriter`

Reply:
[Tis the default](https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/JsonWriter.cs#L349) either way :)

Reply:
Defaults can change, and might not be noticed?

Reply:
For sure - haven't seen it being used anywhere else granted (including in the Newtonsoft docs), so I'm good leaving as-is.